certbot-nginx/certbot_nginx/obj.py

```diff
@@ -96,6 +96,11 @@ def __repr__(self):
     def super_eq(self, other):
```

Comment:
Previously, this method just called the equal method of this class' parent, which is why it was glossed `super_eq`. If the address check is going here, could you rename it to "raw address eq" or something of the sort?